### ISSUE TITLE
Error when no api key is set

### ISSUE THREAD

**Issue by tempo22** [adopted-ember-addons/ember-cli-bugsnag#107]
[This line](https://github.com/adopted-ember-addons/ember-cli-bugsnag/blob/5f94b84652a1589c2a4f1c177a7b582e90e8e6bc/app/instance-initializers/bugsnag.js#L13) will issue the following error if we don't set an api key:
```
Uncaught TypeError: Cannot read property '_config' of null
    at Object.initialize (bugsnag.js:15)
```

What do you think of improving the guard like this ?
```
  if (!Bugsnag._client || Bugsnag._client._config.apiKey === undefined) {
    return;
  }
```

**Comment by jderr-mx** [comment]
@tempo22 this looks reasonable, would you like to submit a PR?

**Comment by bertdeblock** [comment]
Would it be possible to release this fix?

**Comment by jderr-mx** [comment]
@bertdeblock, I thought that I had released it already. My apologies, I will release it today.

**Comment by bertdeblock** [comment]
Not a problem, thank you!

**Comment by jderr-mx** [comment]
This is released in v3.0.1
